src/screens/OtherProfile/index.js

```diff
@@ -302,7 +297,6 @@ const OtherProfile = () => {
             buttonStyle={{paddingLeft: 0}}
             onPress={() => {
               updateFollowDiscoveryContext(false, {user_id: other_id});
-              setIsCurrentFollow((v) => !v);
               handleSetUnFollow();
             }}>
             <View style={styles.buttonFollowing(isAnonimity)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the state update `setIsCurrentFollow((v) => !v);` is a good move since this state has been replaced by a function call. This simplifies the component and removes unnecessary re-renders.
```diff
- setIsCurrentFollow((v) => !v);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->